app/com/linkedin/drelephant/tuning/engine/SparkConfigurationConstants.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.drelephant.tuning.engine;
+
+public class SparkConfigurationConstants {
+  public final static String SPARK_EXECUTOR_MEMORY_KEY = "spark.executor.memory";
+  public final static String SPARK_MEMORY_FRACTION_KEY = "spark.memory.fraction";
+  public final static String SPARK_DRIVER_MEMORY_KEY = "spark.driver.memory";
+  public final static String SPARK_EXECUTOR_INSTANCES_KEY = "spark.executor.instances";
+  public final static String SPARK_EXECUTOR_CORES_KEY = "spark.executor.cores";
+  public final static String SPARK_SERIALIZER_KEY = "spark.serializer";
```

Comment:
Few of these constants are unused. Are they required?

Reply:
They might be required in future